README.md

```diff
@@ -1,33 +1,155 @@
+The SPDX standard helps facilitate compliance with free and open source
+software licenses by standardizing the way license information is shared across
+the software supply chain. SPDX reduces redundant work by providing a common
+format for companies and communities to share important data about software
+licenses and copyrights, thereby streamlining and improving compliance.
 
 This repository holds under active development version of the specification as:
 
-* [MarkDown](https://github.com/spdx/spdx-spec/tree/master/chapters) (`master` branch)
-* HTML (gh-pages branch, built on every commit to `master` and `development/` branches)
-  * [Current](https://spdx.github.io/spdx-spec/v3.0/)
+- [Markdown](https://github.com/spdx/spdx-spec/tree/development/v3.0.1/docs)
+  (`development/v3.0.1` branch)
+- HTML (`gh-pages` branch, built on every commit to `master` and `development/`
+  branches)
+  - Current: [https://spdx.github.io/spdx-spec/v3.0/](https://spdx.github.io/spdx-spec/v3.0/)
 
-See for the official [releases of the specification](https://spdx.org/specifications) or additional information also the [SPDX website](https://spdx.org).
+The model itself is under active development at [spdx/spdx-3-model](https://github.com/spdx/spdx-3-model/)
+repo (`main` branch).
 
-## Specification Structure
+See for the official
+[releases of the specification](https://spdx.org/specifications)
+or additional information also the [SPDX website](https://spdx.org).
 
-The specification consists of a model which is generated from the [spdx-3-model](https://github.com/spdx/spdx-3-model) repository and additional information in the `docs` directory.
+## Specification structure
 
-The `examples` directory contains examples of various SPDX serializations for the current version of the spec.
+This repository consists of these files and directories:
 
-# Building the specification
+- `bin/` - Scripts for spec generation.
+- `docs/` - Additional documentations:
+  - `annexes/` - Annexes for the specification.
+  - `css/` - Style sheets for HTML.
+  - `images/` - Model diagrams. These files are to be generated from a diagram
+    file [model.drawio](https://github.com/spdx/spdx-3-model/blob/main/model.drawio)
+    in `spdx/spdx-3-model` repo and manually copied here.
+  - `licenses/` - Licenses that used by SPDX specifications.
+  - `model/` - Model files. *This subdirectory _is to be created_ by a script
+    from `spdx/spec-parser` repo, using model information from
+    `spdx/spdx-3-model` repo (see the build instructions below).
+- `examples/` - Examples of various SPDX serializations for the current version
+  of the spec.
+- `ontology` - Ontology files. Generated by a script from `spdx/spec-parser`
```

Comment:
This is gone now

Reply:
Thank you. I will remove the `ontology` directory from the list.